resources/views/components/frameworks/bootstrap5/pagination.blade.php

```diff
@@ -1,48 +1,73 @@
-<div>
-    @if ($paginator->hasPages())
-        <nav>
-            <ul class="pagination">
-                {{-- Previous Page Link --}}
-                @if ($paginator->onFirstPage())
-                    <li class="page-item disabled" aria-disabled="true" aria-label="@lang('pagination.previous')">
-                        <span class="page-link" aria-hidden="true">&lsaquo;</span>
-                    </li>
-                @else
-                    <li class="page-item">
-                        <button type="button" dusk="previousPage" class="page-link" wire:click="previousPage" wire:loading.attr="disabled" rel="prev" aria-label="@lang('pagination.previous')">&lsaquo;</button>
-                    </li>
-                @endif
-
-                {{-- Pagination Elements --}}
-                @foreach ($elements as $element)
-                    {{-- "Three Dots" Separator --}}
-                    @if (is_string($element))
-                        <li class="page-item disabled" aria-disabled="true"><span class="page-link">{{ $element }}</span></li>
-                    @endif
+<div class="d-flex align-items-center justify-content-between">
+@if($recordCount === 'full')
+    <small class="text-muted d-block">
+        {{ trans('livewire-powergrid::datatable.pagination.showing') }}
+        <span class="font-semibold">{{ $paginator->firstItem() }}</span>
+        {{ trans('livewire-powergrid::datatable.pagination.to') }}
+        <span class="font-semibold">{{ $paginator->lastItem() }}</span>
+        {{ trans('livewire-powergrid::datatable.pagination.of') }}
+        <span class="font-semibold">{{ $paginator->total() }}</span>
+        {{ trans('livewire-powergrid::datatable.pagination.results') }}
+    </small>
+@elseif($recordCount === 'short')
+    <small class="text-muted d-block">
+        <span class="font-semibold"> {{ $paginator->firstItem() }}</span>
+        -
+        <span class="font-semibold"> {{ $paginator->lastItem() }}</span>
+        |
+        <span class="font-semibold"> {{ $data->links()->paginator->total() }}</span>
```

Comment:
$data variable is not available here

```suggestion
        <span class="font-semibold"> {{ $paginator->total() }}</span>
```